### ISSUE TITLE
Maggi deliverables from 1-on-1 meeting Sept 5

### ISSUE THREAD

**Issue by MICOlab-USF** [MICOlab-USF/antarctica-diel#3]
- [x] convert salmon quant files to single data frame
- [x] make and share metadata file with information about samples
- [x] share Arianna's paper on her assembly, annotation, mapping pipeline
- [x] share papers about diel metatranscriptomics in marine ecosystems 
- [x] email joe about the office space

**Comment by MICOlab-USF** [comment]
## papers to check out on metatranscriptomics and diel changes in phytoplankton

- https://journals.asm.org/doi/10.1128/msystems.00896-20
- https://www.nature.com/articles/s41559-021-01606-w
- https://www.frontiersin.org/articles/10.3389/fmars.2022.798877/full
- https://www.nature.com/articles/s41396-020-00793-x

## papers related to mixotrophy-specific genes

- https://www.pnas.org/doi/full/10.1073/pnas.2100916119
- https://www.frontiersin.org/articles/10.3389/fmars.2018.00273/full
- https://academic.oup.com/femsec/article/92/7/fiw106/2470007

## paper not specifically about diel cycling, but about time-series meta-omics in the same region that we are working

- https://www.frontiersin.org/articles/10.3389/fmars.2021.593566/full

## Arianna's papers on Eukrhythmic (assembly, functional annotation, read mapping pipeline) and Eukulele (taxonomic annotation)

- **Eukrythmic:** https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-022-05121-y
- **Eukulele:** https://joss.theoj.org/papers/10.21105/joss.02817

**Comment by MICOlab-USF** [comment]
I merged all the salmon quant files into a single `.csv` file. I also filtered this `.csv` file to remove rows for contigs that did not recruit a total of 5 reads from the dataset. The file is still large, but I am able to load it into Rstudio. Let me know if you are able to load these data files. If they are still too cumbersome, we can think about further data reduction. 

new data files are here: https://drive.google.com/drive/folders/1OZaWdgTDD8HxwDKVGpNtUYk3bD2ROiz8?usp=drive_link
